scripts/mv-pack.js

```diff
@@ -9,14 +9,17 @@
  *
  */
 
+const childProcess = require("child_process");
 const fs = require("fs");
 const path = require("path");
 const packageName = process.argv[2];
 const extension = process.argv[3];
 const fullPackageName = `${packageName}-${process.env.npm_package_version}.${extension}`;
 let targetPath = path.join("..", "..", "dist", fullPackageName);
-if (process.env.GITHUB_ACTIONS) {
-    targetPath = targetPath.replace("SNAPSHOT", `${process.env.GITHUB_REF_NAME}.${process.env.GITHUB_SHA.slice(0, 7)}`);
+if (targetPath.includes("-SNAPSHOT")) {
+    const gitBranch = childProcess.execSync("git rev-parse --abbrev-ref HEAD").toString().trim().replace(/\//g, "_");
+    const gitSha = process.env.CI && childProcess.execSync("git rev-parse --short HEAD").toString().trim();
+    targetPath = targetPath.replace("-SNAPSHOT", gitSha ? `-${gitBranch}.${gitSha}` : `-${gitBranch}`);
```

Comment:
will it be a problem for the `main` branch (after this is merged) ?
IDK if someone is expecting to have a `-snapshot.vsix` vs a `main-<hash>.vsix`

I like the idea either way